### PR TITLE
Namespace for Controlled Vocabularies

### PR DESCRIPTION
**Pull request by pgwillia**
## Context

We're going to be adding digitization which will have it's own controlled vocabulary.  First we move everything ERA to it's own namespace.

Related to #2118

## What's New
1.  Changed API

From `translate_uri(vocab, uri)`

To `translate_uri(namespace, vocab, uri)`

And

From `humanize_uri(vocab, uri)`

To `humanize_uri(namespace, vocab, uri)`

And

From `validates :languages, presence: true, uri: { in_vocabulary: :language }` or   `validates :license, uri: { namespace: :era, in_vocabularies: [:license, :old_license] }`

To `validates :languages, presence: true, uri: { namespace: :era, in_vocabulary: :language }` or   `validates :license, uri: { namespace: :era, in_vocabularies: [:license, :old_license] }`

2. Added `:era` namespace
* config/controlled_vocabularies
* config/locales/en.yml#controlled_vocabularies

### CODE REVIEW

**Review by murny — DISMISSED** [review]
Nice change 👍 

gif

**Comment by pgwillia** [comment]
I didn't have the latest integration.  Tests passing post rebase and resolving conflicts.

**Review by murny — DISMISSED** [review]

File: app/validators/uri_validator.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class URIValidator < ActiveModel::EachValidator
 
   def validate_each(record, attr, value)
+    namespace = options[:namespace]
```
Comment (pgwillia) [inline app/validators/uri_validator.rb:4]:
```suggestion
    namespace = options[:namespace]
    raise ArgumentError, "Namespace not found: #{namespace}" unless CONTROLLED_VOCABULARIES.key?(namespace)
```
```suggestion
    namespace = options[:namespace]
```

Comment (mbarnett) [reply app/validators/uri_validator.rb:4]: I'm thinking once you've made this change, ping me and I'm good to sign off on this PR. Then I'll use it as the base to make some API changes I've got in mind for dealing with https://github.com/ualbertalib/jupiter/pull/2089#discussion_r582400138, and then I'll hand that back to you for you to use with what we discussed in https://github.com/ualbertalib/jupiter/pull/2188

**Review by mbarnett — APPROVED** [review]
Alright, let's merge this and then I'm going to start a branch based on this and tweak the Controlled Vocab API/internals a bit to accommodate the idea of non-localized URIs

**Pull request merged**